### PR TITLE
add capability to accept updates via command option

### PR DESCRIPTION
**Pull request by lalop**
see #4108

### CODE REVIEW

**Comment by lalop** [comment]
the `yes` option is define in https://github.com/hechoendrupal/drupal-console-core/blob/master/src/Application.php#L429-L436

I only patch this command but we can image adding this feature in the confirm method ?

**Comment by lalop** [comment]
As seen with @jmolivas I added a micro-patch to remove dead code in the same command.

**Comment by lalop** [comment]
@enzolutions any chance to see this merge soon ?

**Comment by enzolutions** [comment]
@lalop we will try to review to include in our next release 1.9.1 in a week or two. We have been putting our effort to migrate the documentation and website of Drupal Console to Gatsby https://drupal-console.web.app/docs/fa-ir/chains/what-is-a-chain-command

File: src/Command/Update/ExecuteCommand.php [hunk 1]
```diff
@@ -161,7 +158,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $updateList = update_get_update_list();
             $this->showUpdateTable($this->module === 'all' ?  $updateList: $updateList[$this->module], $this->trans('commands.update.execute.messages.pending-updates'));
 
-            $allowUpdate = $this->getIo()->confirm(
+            $allowUpdate = $assumeYes || $this->$this->getIo()->confirm(
```
[the remark this answers is not in the document]
Comment (lalop) [reply src/Command/Update/ExecuteCommand.php]: and this will stay in git for ever :)

[the remark this answers is not in the document]
Comment (lalop) [reply src/Command/Update/ExecuteCommand.php]: @harold20 good point, sorry for that

**Comment by enzolutions** [comment]
Thanks @lalop

**Pull request merged**